app/internal/page/data.go

```diff
@@ -81,6 +93,7 @@ type Lpa struct {
 	WantToApplyForLpa                           bool
 	CPWitnessCodeValidated                      bool
 	Submitted                                   time.Time
+	Progress                                    Progress
```

Comment:
I was thinking we shouldn't actually store any new data for this, and just tick "LPA signed" if the `Submitted` time is set, since we probably won't end up storing it like this 🔮

Reply:
Loving the mystic meg vibes 😂 good point - I'll drop from `Lpa`